python/orca/src/bigdl/orca/learn/tf2/pyspark_estimator.py

```diff
@@ -353,35 +339,28 @@ def load_weights(self, filepath, by_name=False):
         """
         model = self.model_creator(self.config)
         model.load_weights(filepath, by_name)
-        self.model_weights = model.get_weights()
+        state = {}
+        state["weights"] = model.get_weights()
+        save_pkl(state, os.path.join(self.model_dir, self.application_id + "_state.pkl"))
```

Comment:
It seems the logic to formatting the path occur in multiple places, shall we provide a function like `get_state_path`